projects/ngx-sub-form/src/lib/ngx-sub-form-utils.ts

```diff
@@ -56,7 +56,7 @@ export interface TypedFormGroup<TValue> extends FormGroup {
 export interface TypedFormArray<TValue extends any[]> extends FormArray {
```

Comment:
I wonder actually why we've got `TypedFormArray<TValue extends any[]>` in the first place :thinking: 

I'd rather prefer to have:

```ts
export interface TypedFormArray<TValue> extends FormArray {
  value: TValue[];
  valueChanges: Observable<TValue[]>;
  controls: TypedAbstractControl<TValue>[];
  setValue(value: TValue[], options?: Parameters<FormArray['setValue']>[1]): void;
  patchValue(value: TValue[], options?: Parameters<FormArray['patchValue']>[1]): void;
  getRawValue(): TValue[];
}
```

But that'd be a breaking change if we expose `TypedFormArray` (which I think we do :thinking:)